src/github/graphql.ts

```diff
@@ -1,18 +1,19 @@
-module.exports = addGraphQL
+import { OctokitWithPagination, Variables } from './'
 
-function addGraphQL (octokit) {
+export const addGraphQL = function (octokit) {
   octokit.query = query.bind(null, octokit)
```

Comment:
because you're binding this on octokit, whenever I try to use `context.github.query`, typescript complains because `query` doesn't exist on the type `GithubAPI`. You should augment the typescript definition of the GithubApi to add this, or `context.github` should maybe be of type `GithubApiWithGraphQL` or something.

Reply:
Not sure why this is showed outdated here in the PR.  Definitely still an issue.